### ISSUE TITLE
Feature request : send entire folder

### ISSUE THREAD

**Issue by arnoulfos** [localsend/localsend#554]
### Is your feature request related to a problem? Please describe.

Hi it would be very useful the possibility to select a folder to send it as a whole 

### Describe the solution you'd like

Select a folder to send all its content

**Comment by Tienisto** [comment]
This should be already possible since 1.9.0

**Comment by arnoulfos** [comment]
I would like to be able to send a folder with  subfolders and subsubfolders ... and that the hierarchy of folders would be recreated at the arrival. I don t have the feeling that it is currently possible. I created a folder test with a subfolder subtest and a file in the subfolders and I am getting an error sending it.

Jul 4, 2023 00:04:35 Tien Do Nam ***@***.***>:

**Comment by Tienisto** [comment]
I guess you need at least one file. Empty folders are currently not sent

**Comment by RavenMacDaddy** [comment]
I think what the OP means is currently, the app is simply selecting the files of the folder individually - not the actual structure of the selected folder and subfolders.

For example, let's say the folder selected has the following structure:

Music
- FLAC
-- Progressive house
-- Rock
- MP3
-- Pop

It would be very beneficial if selecting the Music folder kept this structure - rather than sending every file to the selected path in settings as they're.

**Comment by Tienisto** [comment]
Yes, we are speaking of the same issue I guess. LocalSend is file based like git so you need to have a dummy file (e.g. `.keep` or something similar) to make sure that even empty folders are sent.

I will check if a workaround can be implemented as the protocol only support files.
